.circleci/config.yml

```diff
@@ -1209,6 +1217,34 @@ jobs:
             kurtosis engine status || true
 
             echo "Kurtosis setup complete"
+      # Start our devnet
+      - run:
+          name: Start devnet (<<parameters.devnet>>)
+          working_directory: kurtosis-devnet
+          command: |
+            echo "Starting <<parameters.devnet>> devnet..."
+            just <<parameters.devnet>>-devnet
+
+            # Check if the devnet started successfully
+            if ! kurtosis enclave ls | grep -q "<<parameters.devnet>>-devnet"; then
+              echo "Failed to start <<parameters.devnet>> devnet"
+              exit 1
+            fi
+
+            echo "<<parameters.devnet>> devnet started successfully"
+
+            # Wait a moment for the devnet to stabilize
+            sleep 5
```

Comment:
Isn't the non-zero exit code from `just` going to fail the pipeline? It seems like the `exit 1` here will never happen since CI runs have `-e pipefail` set so they exit on exit code 1 from any command.

And I would drop the `sleep 5`, we have health checks in `optimism-package` that will ensure that when `kurtosis run` exits, the system is ready to use.

Reply:
I added that extra check to be extra safe, so we don't rely too much on the kurtosis-devnet justfile semantics. Anyway, I've removed it for simplification as a result of your comment.

The sleep is gone as well.